nodes/default_node.go

```diff
@@ -30,6 +30,8 @@ type DefaultNode struct {
 	Mgmt             *types.MgmtNet
 	Runtime          runtime.ContainerRuntime
 	HostRequirements *types.HostRequirements
+	// Indicates that the node should not start without no license file defined
+	LicenseRequired bool
```

Comment:
@steiler 
maybe we should have a string field `LicensePolicy` with values

* "" - not required
* "required" - fail with an error saying license is required
* "warn" - emit a warning if not specified

"warn" - is for SR OS case, where we allow users to run it without a license, assuming they have it embedded in the image.

Reply:
checkout the new commit, please.